frontend_lib/src/helper.js

```diff
@@ -387,6 +387,27 @@ export const updateTLMAuthor = author => {
     }
 }
 
+export const updateTLMUser = user => {
+  return user
+    ? { ...user, is_from_system_admin: false }
+    : {
+      allowed_space: 0,
+      auth_type: 'internal',
+      avatar_url: null,
+      created: '',
+      email: '',
+      is_active: true,
+      is_deleted: false,
+      is_from_system_admin: true,
+      lang: 'en',
+      profile: 'users',
+      public_name: i18n.t('Unknown'),
+      timezone: '',
+      user_id: 0,
+      username: ''
+    }
```

Comment:
It seems the only difference from updateTLMAuthor is the public name. Would it be possible to avoid the copy paste by using a parameter in updateTLMAuthor to use "Unknown" instead of "System Administrator"?

Reply:
For me the semantics are different between the two, I would prefer to leave it like this, but it's not such a strong preference so if you really want I can change it.